randomname/util.py

```diff
@@ -188,7 +189,9 @@ def doalias(fname):
 
 def safepath(f):
```

Comment:
@beasteers This function is actually a bit problematic as it in some cases gets file paths and in some cases it gets names specified by the users. On unix that's not a problem as we use `/` in both cases but on Windows it could be problematic. 

Why do we need this function in the first place? If users specify a name starting with `..` then we'd simply not match this to a word list file, don't we?

Reply:
This is just about keeping paths inside a certain directory. I think this would be preferrable:

`os.path.abspath(os.sep + f.replace('/', os.sep)).lstrip(os.sep)`

Reply:
> This is just about keeping paths inside a certain directory.

In which cases can it happen that we don't stay inside the same directory?